### PR TITLE
[WIP] Implement new serial console in s390 backend

### PR DESCRIPTION
**Pull request by Soulofdestiny**

### CODE REVIEW

File: consoles/sshIucvconn.pm [hunk 1]
```diff
@@ -35,8 +35,9 @@ sub activate {
     my ($self) = @_;
 
     my $hostname = $self->{args}->{hostname};
+    print "SSH to $hostname\n";
     my $chan = $self->backend->start_ssh_serial(hostname => $hostname, password => $self->{args}->{password}, username => 'root');
-    $chan->exec("iucvconn $hostname hvc0");
+    while (1) { $chan->exec("iucvconn LINUX156 lnxhvc0"); }
```
Comment (Soulofdestiny) [inline consoles/sshIucvconn.pm]: this needs to be adjusted to use the correct guest LINUX### - hardcoded by now

File: consoles/sshIucvconn.pm [hunk 2]
```diff
@@ -35,8 +35,9 @@ sub activate {
     my ($self) = @_;
 
     my $hostname = $self->{args}->{hostname};
+    print "SSH to $hostname\n";
     my $chan = $self->backend->start_ssh_serial(hostname => $hostname, password => $self->{args}->{password}, username => 'root');
-    $chan->exec("iucvconn $hostname hvc0");
+    $chan->exec("iucvconn LINUX156 lnxhvc0");
```
Comment (Soulofdestiny) [inline consoles/sshIucvconn.pm]: to run this in a while(1) is not working properly,  needs further investigation
still TODO: use hostname as a variable and not hardcoded

File: backend/s390x.pm [hunk 3]
```diff
@@ -56,8 +56,6 @@ sub do_start_vm {
 sub do_stop_vm {
     my ($self) = @_;
 
-    $self->stop_serial_grab;
```
Comment (coolo) [inline backend/s390x.pm]: why this? This sounds implausible

Comment (Soulofdestiny) [inline backend/s390x.pm]: probably some fragment of trial and error...  fixed

**Pull request merged**